apps/coscrad-frontend/src/components/live/live.tsx

```diff
@@ -0,0 +1,12 @@
+import { useContext } from 'react';
+import { ConfigurableContentContext } from '../../configurable-front-matter/configurable-content-provider';
+
+export interface AboutProps {
+    about: string;
+}
+
+export const Live = (): JSX.Element => {
```

Comment:
I'm not a big fan of the name `Live`. How about `LiveStream` or `LiveRadioStream`?

Reply:
we fixed this